uber/site_sections/preregistration.py

```diff
@@ -191,7 +191,8 @@ def form(self, session, message='', edit_id=None, **params):
                     if group.badges:
                         Tracking.track(track_type, group)
 
```

Comment:
Another question: since this is in our ``preregistration`` section where someone is registering for the first time, there's no possible way that this attendee could already be in our database, right?  So why the ``.filter(Attendee.id != attendee.id)`` check?